caravel/assets/javascripts/modules/caravel.js

```diff
@@ -10,6 +10,7 @@ var sourceMap = {
   bar: 'nvd3_vis.js',
   bubble: 'nvd3_vis.js',
   big_number: 'big_number.js',
+  big_number_total: 'big_number.js',
```

Comment:
I feel like the biggest difference between these two is time compare vs not, and that's not immediately obvious with this naming scheme. If we made that distinction ideally the old number view would be called `BigNumberTimeCompare` and the new one would be `BigNumber`. 

I think changing the name of the old/time compare number would break a lot of URLs, though, because `viz_type` would be stale for what is now called `big_number, right @mistercrunch ? so prob not worth the headache.